frontend/resourceadm/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -6,10 +6,14 @@ import { useTranslation } from 'react-i18next';
 type ErrorMessageProps = {
   title: string;
   message: string;
-  children?: React.ReactNode;
+  children?: React.JSX.Element;
```

Comment:
Children of a component should be ``` ReactNode``` 😄 
Only the return type of the component should have the ```React.JSX.Element```